### ISSUE TITLE
iOS update

### ISSUE THREAD

**Issue by SeanatGDF** [growingdatafoundation/growingdatafoundation.github.io#1]
Hi,
I have had a number of enquiries regarding when the iOS update will be available.
I suggest that we make this somewhat of a priority as it has been waited on for quite some time now...

**Comment by Morganiseit** [comment]
Yeah. I have had similar requests. There are a lot of people using iOS. Sad
but true.

- Morgan

**Comment by SeanatGDF** [comment]
;)
